docs/outlook/delay-delivery.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Manage the delivery date and time of a message (preview)
+description: Learn how to get and set the delivery date and time of a message in compose mode.
+ms.date: 04/28/2023
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Manage the delivery date and time of a message (preview)
+
+The Outlook client gives you the option to delay the delivery of a message, but requires you to keep Outlook and your device running to send it at the specified time. With the Office JavaScript API, you can now implement an Outlook add-in that sends scheduled messages even with your Outlook client closed or with your device turned off. This capability provides your users with the convenience to schedule email marketing campaigns or time a message to be delivered during a colleague or customer's business hours.
+
+> [!IMPORTANT]
+> Features in preview shouldn't be used in production add-ins. We invite you to try out this feature in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
+
+## Configure the manifest
+
+To schedule the delivery of a message, your add-in must be able to activate in message compose mode. This is defined through the [MessageComposeCommandSurface](/javascript/api/manifest/extensionpoint#messagecomposecommandsurface) extension point in an XML manifest or the **mailCompose** "contexts" property in a [Unified Microsoft 365 manifest (preview)](../develop/json-manifest-overview.md).
+
+For further guidance on how to configure an Outlook add-in manifest, see [Outlook add-in manifests](manifests.md).
+
+## Access the delivery property of a message
+
+The [item.delayDeliveryTime](/javascript/api/outlook/office.messagecompose?view=outlook-js-preview&preserve-view=true#outlook-office-messagecompose-delaydeliverytime-member) property returns a [DelayDeliveryTime](/javascript/api/outlook/office.delaydeliverytime?view=outlook-js-preview&preserve-view=true) object that provides you with methods to get or set the delivery date and time of a message.
+
+## Get the delivery date and time of a message
+
+To get the delivery date and time of a message in compose mode, call [item.delayDeliveryTime.getAsync](/javascript/api/outlook/office.delaydeliverytime?view=outlook-js-preview&preserve-view=true#outlook-office-delaydeliverytime-getasync-member(1)) as shown in the following example. If a delivery date hasn't been set on a message yet, the call returns `0`. Otherwise, it returns a [JavaScript Date object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Date).
+
+```javascript
+// Gets the delivery date and time of a message.
+Office.context.mailbox.item.delayDeliveryTime.getAsync((asyncResult) => {
```

Comment:
seems like you're missing closing braces/parentheses from this?

Reply:
😱 thanks for catching that!